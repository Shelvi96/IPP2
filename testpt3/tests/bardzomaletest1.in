NEW ID

;;61>26697:225
14>254080
4;;;9>29
7161;782>3764;668:;
2712;4>864;
:9:98;;6>646::
6033734>284:691;
328;>89
629445>99644;56;9
6:1954;68>6
7038982053>9
921131>228;
05;4>844401
2>95;:
14927:5>511456
2676:;:331>0:7:302
6>3;385
@2;;89:2;:12162289432;40
@;04791330601420276858:2965
@514:087;5820080153572;38;520
@74118054;517;1;06
@5
@943003;55;15336333;68450:404
@955185548317463
@4628:882180908511:72706337
@866724:226;90;994639;54159;8
@;3059876:2081984;:095:8:2
@731440;89
@857677305;849952
@155309985901
@1:48433;70;780417774
@28:;931980:6863312:6556:28
@97129109:72;4292:79114
