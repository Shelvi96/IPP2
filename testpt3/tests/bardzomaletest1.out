22020101000
231948935195904
1121000000000
4104
0
2101120000000000
136
75322943766169
33400646668210176
19329077932992
0
2202
0
64
22011000000000
150994944
